extensions/wikia/Email/templates/main.mustache

```diff
@@ -5,14 +5,21 @@
 <body>
 
 <div class="main-wrapper">
+	<div class="fandom-bar">
+		<div class="fandom-color-1"></div><!--
```

Comment:
Yes they are needed in order for some email clients to not add additional space between bars in header.